### ISSUE TITLE
[feature] Could we only bump versions and make tags for some packages ?

### ISSUE THREAD

**Issue by awxalbert** [changesets/changesets#880]
## Affected Packages

`@changesets/cli`

## Problem

I use monorepo structure suggested by [Turborepo](https://turborepo.org/) like this:

```shell
--apps
  |--app1
  |--app2
--packages
  |--ui
```
app1 and app2 are just some SPAs, not packages. However, changesets will attempt to publish every recognized packages which is not necessary in this case.

## Proposed solution

Maybe provide a flag just to bump versions in package.json and make git tags, skipping the publish step. I think this kind of use case is pretty common nowadays.

**Comment by jsinclair-indeed** [comment]
@awxalbert do your app's package.json have `private: true` set?

I might be misunderstanding your problem, but if you want to stop them being published that should work as per [docs/config-file-options.md#ignore-array-of-packages](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#ignore-array-of-packages).

**Comment by Andarist** [comment]
Yeah, in this case, the `private: true` seems like the most straightforward solution unless there are some other constraints for the given use case.

**Comment by steve-taylor** [comment]
It would be nice to be able to ignore version bumping specific packages on the command line.

I'm also working in a monorepo with multiple apps (already `"private": true`) and publishable packages. When I version bump app1 in preparation for releasing an update of that app, it doesn't mean I'm ready to version bump app2 and app3.

Something like this would be nice:

```sh
changeset version --exclude @mycompany/app2 --exclude @mycompany/app3
```

It would be a slightly complex feature which would involve:
* Splitting changesets
* Refusing to exclude version bumping dependencies of changed packages

**Comment by Andarist** [comment]
Yeah, I'm glad that you see the complexity involved in this :p It's definitely doable but I'm afraid that I don't have manpower to implement this right now on my own. I would happily discuss the API, implementation details and review a PR though

**Comment by awxalbert** [comment]
@Andarist @jsinclair-indeed Thanks for your reply. However, `private: true` doesn't resolve my problem, the original intention was to prevent changeset from publishing **private apps** because it's not necessary, I only need tags during a publish phase. 

But now I found my problem is resolved in this release: [@changesets/cli@2.25.0](https://github.com/changesets/changesets/blob/main/packages/cli/CHANGELOG.md#2250), and it's documented [here](https://github.com/changesets/changesets/blob/main/docs/versioning-apps.md), it's exactly what I need, I'll close this issue.
